rt/cpm/cowgol.coh

```diff
@@ -18,6 +18,14 @@ sub AlignUp(in: intptr): (out: intptr) is
 	out := in;
 end sub;
 
+# Returns a = l = character
+# So we don't need mov l, a
+sub get_char(): (c: uint8) is
+	@asm "mvi c, 1";
+	@asm "call 5";
+	@asm "shld", c;
```

Comment:
This will write L to `c`, but will write H to the address after `c`, potentially corrupting something. Luckily the BDOS returns the character in A too so a simple `lda` instead of `shld` should work here.

Reply:
Aha. Fixed with a force push.

Reply:
Looks good --- thanks!